LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -4666,3 +4666,281 @@ TacticalText="<Bullet/> Плазмобластер выпускает разру
+
+[MissDamageUpgrade_Bsc X2WeaponUpgradeTemplate]
+FriendlyName="Рукоятка"
+FriendlyNamePlural="Рукоятки"
+BriefSummary="Промахи из основого оружия с 20% шансом могут стать царапиной."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Adv X2WeaponUpgradeTemplate]
+FriendlyName="Улучшенная рукоятка"
+FriendlyNamePlural="Улучшенные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 25% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Sup X2WeaponUpgradeTemplate]
+FriendlyName="Элитная рукоятка"
+FriendlyNamePlural="Элитные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 30% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+; End Translation
+
+;LWOTC translated
+
+[MissDamageUpgrade_Bsc X2WeaponUpgradeTemplate]
+FriendlyName="рукоятка"
+FriendlyNamePlural="Рукоятки"
+BriefSummary="Промахи из основого оружия с 20% шансом могут стать царапиной."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Adv X2WeaponUpgradeTemplate]
+FriendlyName="Улучшенная рукоятка"
+FriendlyNamePlural="Улучшенные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 25% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Sup X2WeaponUpgradeTemplate]
+FriendlyName="Элитная рукоятка"
+FriendlyNamePlural="Элитные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 30% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+; End Translated
+
+;LWOTC needs translation
+
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%."
+
+[CritUpgrade_Adv X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%."
+
+; End Translation
+
+
+; LWOTC needs translation
+BriefSummary="Увеличивает шанс крита на +<Ability:CRITINCREASE/>%."
+; End translation
+
+
+;LWOTC needs Translation
+
+[BasicStockProject X2TechTemplate]
+DisplayName="Приклад"
+Summary="Мы можем разобрать и реконструировать приклад для массового производства. Любой оперативник с оружием оборудованным прикладом имеет больше шансов попасть царапиной"
+
+[AdvancedStockProject X2TechTemplate]
+DisplayName="Улучшенные приклады"
+Summary="Позволяет массово производить улучшенные приклады."
+
+[SuperiorStockProject X2TechTemplate]
+DisplayName="Элитные приклады"
+Summary="Позволяет производить элитные приклады, которые ещё больше увеличивают шансы поцарапать при промахе."
+
+;End Translation
```

Comment:
You should be modifying the existing entries. Follow these steps:

1. Find the heading in *XComGame.rus*, e.g. `[BasicStockProject X2TechTemplate]`
2. Change the existing *values* (the text in quotes after `=`) to the new translation

Reply:
It also the same as original lw2

Reply:
deleted